ManagedWinapi/Accessibility/AccessibleObjectListener.cs

```diff
@@ -292,6 +318,32 @@ public SystemAccessibleObject AccessibleObject
         }
     }
 
+
+    /// <summary>
+    /// This enumeration lists Flag values that specify the location of the hook function and of the events to be skipped.
+    /// </summary>
+    [Flags]
+    public enum AccessibleEventFlags
+    {
+
+        /// <summary>
+        /// The callback function is not mapped into the address space of the process that generates the event.
+        /// </summary>
+        OUTOFCONTEXT = 0x0000,
+        /// <summary>
+        /// Prevents this instance of the hook from receiving the events that are generated by the thread that is registering this hook.
+        /// </summary>
+        SKIPOWNTHREAD = 0x0001,
+        /// <summary>
+        /// Prevents this instance of the hook from receiving the events that are generated by threads in this process. This flag does not prevent threads from generating events.
+        /// </summary>
+        SKIPOWNPROCESS = 0x0002,
+        /// <summary>
+        /// The DLL that contains the callback function is mapped into the address space of the process that generates the event.
+        /// </summary>
+        INCONTEXT = 0x0004

```

Comment:
Thanks for the pull request :)

What happens if you specify `INCONTEXT` but the application that created the event is not a .NET application (or a .NET application that uses a different CLR, i. e. 1.x vs 2.0/3.x vs 4.x)?

If that causes the target application to crash (or the hooking to fail), I'd rather not allow this flag.

By the way, may I ask what is your use case for these flags? I'm used to pull requests with a bit more details :)